packages/app/lib/ui/widgets/species_uses_aggregate.dart

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+import 'package:app/ui/widgets/pagination_list.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter/material.dart';
+
+import 'package:app/io/p2panda/publish.dart';
+import 'package:app/models/sightings.dart';
+import 'package:app/models/used_for.dart';
+import 'package:app/ui/widgets/tag_item.dart';
+import 'package:app/ui/widgets/used_for_dedup_tag_list.dart';
+import 'package:app/ui/widgets/card.dart';
+import 'package:app/ui/widgets/card_header.dart';
+
+class SpeciesUsesAggregate extends StatelessWidget {
+  final DocumentId id;
+
+  const SpeciesUsesAggregate({super.key, required this.id});
+
+  @override
+  Widget build(BuildContext context) {
+    return MeliCard(
+      child: Column(
+        children: [
+          MeliCardHeader(
+            title: AppLocalizations.of(context)!.usedForCardTitle,
+          ),
+          SpeciesUsesAggregateList(id: id),
+        ],
+      ),
+    );
+  }
+}
+
+class SpeciesUsesAggregateList extends StatelessWidget {
+  const SpeciesUsesAggregateList({
+    super.key,
+    required this.id,
+  });
+
+  final DocumentId id;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      constraints: const BoxConstraints(maxHeight: 300),
+      child: SingleChildScrollView(
+        child: Container(
+            padding:
+                const EdgeInsets.symmetric(vertical: 10.0, horizontal: 18.0),
+            child: FetchAll(
```

Comment:
We use the `FetchAll` widget here to get all sightings for a particular species and then pass the result onto `UsedForTagsList` which.....